### DISCUSSION TITLE
where are the index.html files located for daedalos

### DISCUSSION THREAD

**Discussion by PurpleVoidEpic** [DustinBrett/daedalOS#200] [Q&A] [answered]
I need these files so i can port this html to uwp for my xbox one

**Comment by DustinBrett** [comment, marked as answer]
Those files can be created by cloning the repo and running:

- `yarn` to install the package dependencies
- `yarn build` to build the project
- `yarn export` to create the `out` folder with the html/js required

Another command instead of export can also be `yarn deploy` which runs some additional minification steps on the `out` folder.

**Comment by CorruptionVR** [comment]
hey, how do I run commands like that on github? or is it a terminal only thing?

**Reply by DustinBrett** [reply]
Those are the commands you'd run locally on your machine in the terminal. You could make GitHub Actions to also run these commands. I have one for tests. https://github.com/DustinBrett/daedalOS/blob/main/.github/workflows/main.yml

**Reply by CorruptionVR** [reply]
how do I use github actions for that?

**Reply by DustinBrett** [reply]
You would need to fork the repo to your account and then you could checkout the repo and make your own workflow with the commands you want it to do upon commits.
